config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -42,8 +42,8 @@ presubmits:
       testgrid-tab-name: pull-kueue-verify-main
       description: "Run kueue verify checks"
     spec:
-      containers:
-      - image: golang:1.18
+      containers: 
```

Comment:
this trailing white space -> https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/test-infra/25657/pull-test-infra-verify-lint/1504183906134396928